src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java

```diff
@@ -586,6 +714,29 @@ public SELF doesNotContain(double... values) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual array does not contain the values of the given array.
+   * <p>
+   * Example:
+   * <pre><code class='java'> // assertion will pass
+   * assertThat(new double[] { 1.0, 1.0 }).doesNotContain(new Double[] { 2.0 });
+   *
+   * // assertion will fail
+   * assertThat(new double[] { 1.0, 1.0, 2.0 }).doesNotContain(new Double[] { 2.0 });</code></pre>
+   *
+   * @param values the given values.
+   * @return {@code this} assertion object.
+   * @throws NullPointerException if the given argument is {@code null}.
+   * @throws IllegalArgumentException if the given argument is an empty array.
+   * @throws AssertionError if the actual array is {@code null}.
+   * @throws AssertionError if the actual array contains any of the given values.
+   */
+  public SELF doesNotContain(Double[] values) {
```

Comment:
Please also add

```java
public SELF doesNotContain(Double[] values, Offset<Double> precision)
```

Reply:
https://github.com/assertj/assertj-core/blob/c0f48d3858e047ed0421a36c435e6f1b5503edcb/src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java#L610

It's already there